### PR TITLE
MAT-5084 CMS Id issue

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## PR Template

Jira Ticket: [MAT-5084](https://jira.cms.gov/browse/MAT-5084)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [ ] Required tests are included
* [ ] No extemporaneous files are included (i.e. Complied files or testing results)
* [x] This PR is into the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e. Changing auth methods, Adding a new user type, Adding a required but vulnerable package)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: lambda-function/src/utils/measureConversionUtils.ts [hunk 1]
```diff
@@ -296,3 +298,10 @@ const getSelected = (population: Population, selectedPopulations: Population[]):
   });
   return selected;
 };
+
+const getCmsId = (measureResourceJson: string): string => {
+  const measureResource = JSON.parse(measureResourceJson);
+  const identifier = measureResource.identifier;
```
Comment (adongare) [inline lambda-function/src/utils/measureConversionUtils.ts:304]: slight ambiguous. can we rename this to `identifiers` and then `identifiers.find(..)` or just inline `measureResource.identifier?.find`

Comment (sb-cecilialiu) [reply lambda-function/src/utils/measureConversionUtils.ts]: Just did thanks!

Comment (sb-cecilialiu) [reply lambda-function/src/utils/measureConversionUtils.ts]: @adongare I seem to have vulnerability problems: 
source-map-resolve  >=0.5.1
  Depends on vulnerable versions of decode-uri-component, 
can you please help take a look? Thanks!

Comment (adongare) [reply lambda-function/src/utils/measureConversionUtils.ts]: doesn't look like it has a patch available. @sb-benohe can we ignore it for now? this is dev/test dependency. we'll find alternative to this.

Comment (adongare) [reply lambda-function/src/utils/measureConversionUtils.ts]: nvm, we are good with it now. removed vulnerable package. Thanks, Cecilia.

**Review by adongare — APPROVED** [review]
